package.json

```diff
@@ -28,20 +28,21 @@
   "scripts": {
     "lint": "eslint src",
     "test": "yarn run lint",
-    "release": "yarn semantic-release",
-    "prettier": "prettier --write src/**/*.js"
+    "release": "yarn semantic-release"
   },
   "dependencies": {
-    "prop-types": "^15.7.2",
-    "react-native-modal": "^11.0.2"
+    "prop-types": "^15.7.2"
   },
   "devDependencies": {
     "@semantic-release/git": "^7.0.8",
-    "eslint": "^5.16.0",
-    "eslint-plugin-react-app": "^4.0.1",
+    "eslint": "^6.4.0",
+    "eslint-config-prettier": "^6.3.0",
+    "eslint-plugin-prettier": "^3.1.1",
+    "eslint-plugin-react-app": "^6.0.3",
     "prettier": "^1.16.4",
-    "react": ">=15.0.0",
-    "react-native": ">=0.24.0",
     "semantic-release": "^15.13.3"
+  },
+  "peerDependencies": {
+    "@react-native-community/datetimepicker": ">=1.0.0"
```

Comment:
this caught me out. I'm usually pretty OCD about this stuff, but I have a stack of 15 peerDependency warnings right now owing to various bits of module non-maintenance, so I did not notice that there was a 16th one, and that I needed to manually add this. It isn't mentioned in the readme diff either, so I'd suggest either moving this to dependencies (like the previous one was) or making sure the readme mentions it and the changelog clearly mentions it as a breaking change / must-do-to-migrate

Reply:
You're 100% right. 
Since this is the most important part of the PR I wanted to add it to both the changelog and the README but I haven't done it yet... should have at least told you before pinging you 🤦‍♂ 
Thanks for reporting, will add it to the docs soon (when the CHANGELOG is ready).

Reply:
Please also notice that there are other breaking changes and deprecation (the deprecations are still WIP) in the opening post.